.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -100,15 +100,119 @@ body:
     validations:
       required: true
 
+  - type: markdown
+    attributes:
+      value: |
+        ‎‎‎‎‎‎‎‎
+
+  - type: markdown
+    attributes:
+      value: "### What filters do you have enabled?"
+
+  - type: markdown
+    attributes:
+      value: |
+        Select enabled filters from the list or enter URLs of any custom filters in the comment field below.
+
   - type: dropdown
-    id: filters
+    id: filters_adblock
     attributes:
-      label: What filters do you have enabled?
-      description: Select enabled filters from the list or enter URLs of any custom filters in the comment field below.
+      label: Ad Blocking
       multiple: true
       options:
         - AdGuard Base filter
         - AdGuard Mobile Ads filter
+        - EasyList
+    validations:
+      required: true
+
+  - type: dropdown
+    id: filters_privacy
+    attributes:
+      label: Privacy
+      multiple: true
+      options:
+        - AdGuard URL Tracking filter
+        - AdGuard Tracking Protection filter
+        - EasyPrivacy
+        - Legitimate URL Shortener
+        - Fanboy's Enhanced Tracking List
+        - Peter Lowe's Blocklist
+        - Fanboy's Anti-Facebook List
+        - Fanboy's Anti-thirdparty Fonts
+    validations:
+      required: false
+
+  - type: dropdown
+    id: filters_social
+    attributes:
+      label: Social Widgets
+      multiple: true
+      options:
+        - AdGuard Social Media filter
+        - Fanboy's Social Blocking List
+    validations:
+      required: false
+
+  - type: dropdown
+    id: filters_annoyances
+    attributes:
+      label: Annoyances
+      multiple: true
+      options:
+        - AdGuard Annoyances filter
+        - AdGuard Cookie Notices filter
+        - AdGuard Popups filter
+        - AdGuard Mobile App Banners filter
+        - AdGuard Other Annoyances filter
+        - AdGuard Widgets filter
+        - Adblock Warning Removal List
+        - Fanboy's Annoyances
+        - Web Annoyances Ultralist
+        - I don't care about cookies
+        - EasyList Cookie List
+        - Dandelion Sprout's Annoyances List
+    validations:
+      required: false
+
+  - type: dropdown
+    id: filters_security
+    attributes:
+      label: Security
+      multiple: true
+      options:
+        - Online Malicious URL Blocklist
+        - NoCoin Filter List
+    validations:
+      required: false
+
+  - type: dropdown
+    id: filters_other
+    attributes:
+      label: Other
+      multiple: true
+      options:
+        - Filter unblocking search ads and self-promotion
+        - AdGuard Experimental filter
+    validations:
+      required: false
+
+  - type: dropdown
+    id: filters_dns
+    attributes:
+      label: Security
```

Comment:
There are 2 `Security` sections now.
<details><summary>Screenshot</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/29142494/f984024a-5f5d-43c8-9cb1-94f6f75bbd50)

</details>

Maybe `AdGuard DNS filter` should be moved to `Other` section?

Reply:
It is just a wrong label) Renamed to `DNS filters`